services/sleeper/versioning/integration.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
```

Comment:
not sure the integration test is optimal since it didn't notice the validation output file was empty

Reply:
well the integration test actually does not check the contents. since this is up to the service owner to make sure contents is correct. it should only check that the file is generated if it should be.